docs/book/src/reference/versions.md

```diff
@@ -121,6 +124,16 @@ The Kubeadm Control Plane requires the Kubeadm Bootstrap Provider.
 | v1.1 (v1beta1)  | v1.8.6                          |
 | v1.2 (v1beta1)  | v1.9.2                          |
 
+#### Kubernetes version specific notes
+
+**1.24**:
+
+* Kubeadm Bootstrap Provider:
+    * `kubeadm` now sets both the `node-role.kubernetes.io/control-plane` and `node-role.kubernetes.io/master` taints on control plane nodes.
+    * `kubeadm` now only sets the `node-role.kubernetes.io/control-plane` label on control plane nodes (`node-role.kubernetes.io/master` is not set anymore).
```

Comment:
Does it now set both or only :-)

Reply:
Both taints only the control plane label

Reply:
Added another "label" in the second point to avoid confusion

Reply:
ah, was a bit confusing to find the distinction 👍 

Maybe rephrasing it to have `taints` and `label` not at the end would make it more readable.

But also ok for me as is.

Reply:
Or highlighting tains/label in some way.

Reply:
I would rely now on folks reading it twice if they are confused :)